### DISCUSSION TITLE
The differenct parameter in web.config

### DISCUSSION THREAD

**Discussion by decie** [dotnet/aspnetcore#28871] [Q&A] [answered]
After publishing asp.net core, it will generate web.config. In this file, what is the difference of  processPath=".\MyApplication.exe" or processPath=".\MyApplication.dll".  When to use  which one?

**Comment by gfoidl** [comment, marked as answer]
> processPath=".\MyApplication.dll"

Won't work, as it's not a process -- "just a dll".

> processPath=".\MyApplication.exe"

Is an application -- either self-contained or from .NET Core 3 onwards produced by the SDK, so that on windows you can run this exe instead of `dotnet MyApplication.dll`.

So coming back to the dll:
```xml
...
<aspNetCore processPath="dotnet" arguments=".\MyApplication.dll" ... />
...
```
should be used if you don't want to use the exe.
